### PR TITLE
refactor(core): Use DI for WorkflowRunner (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
continue moving code to DI.

## Review / Merge checklist
- [x] PR title and summary are descriptive

### CODE REVIEW

File: packages/cli/src/WebhookHelpers.ts [hunk 1]
```diff
@@ -590,8 +590,7 @@ export async function executeWebhook(
 		}
 
 		// Start now to run the workflow
-		const workflowRunner = new WorkflowRunner();
```
Comment (krynble) [inline packages/cli/src/WebhookHelpers.ts:593]: Interesting, intrigued to imagine why we wanted to instantiate this every time versus using it as a singleton. Not a major thing, but certainly a performance improvement.

**Review by krynble — DISMISSED** [review]
LGTM, I hope we don't get cornered in yet another dependency cycle at some point; preliminary testing seems all right!

**Comment by krynble** [comment]
Just saw the test results 😐

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.27.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.27.2)

**Pull request merged**